packages/@aws-cdk/aws-sns/lib/subscription.ts

```diff
@@ -16,7 +16,7 @@ export interface SubscriptionProps {
    *
    * The meaning of this value depends on the value for 'protocol'.
    */
-  readonly endpoint: any;
+  readonly endpoint: string;
```

Comment:
Call this out as a BREAKING CHANGE in the commit message that you merge.

Reply:
will do